primes.rb

```diff
@@ -2,5 +2,22 @@ class Primes
   def self.sum_to(limit = 100)
     # TODO - add your prime number solution here...
     "I'm working on it!"
+    return 0 if limit==1 || limit==0
+    return 'The number should be positive' if limit<0
```

Comment:
Most Rubyists don't bother with defensive statements like this - although you're right it should be positive. This could also be handled by using exception handling, which we'll cover